ArmTemplates/ServiceBus/servicebus-topic.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2018-05-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serviceBusNamespaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of an existing Service Bus namespace to add the topic to"
+            }
+        },
+        "serviceBusTopicName": {
+            "type": "string",
+            "metadata": {
+                "description": "Topic name to add to Service Bus"
+            }
+        },
+        "defaultTTL": {

```

Comment:
Maybe rename this to `defaultMessageTtl`  to better indicate what the ttl is applying to?

Reply:
Changed to messageDefaultTTL in line with other change